### DISCUSSION TITLE
LDAP for Teleport login?

### DISCUSSION THREAD

**Discussion by marcosimioni** [gravitational/teleport#11997] [Q&A] [answered]
Hi all,

Is it possible to let Teleport authenticate via an AD (i.e. via LDAP)?

I don't see such authentication method mentioned on https://goteleport.com/docs/setup/reference/authentication/

Any pointer?

**Comment by zmb3** [comment, marked as answer]
You should be able to use AD FS: https://goteleport.com/docs/enterprise/sso/adfs

**Reply by marcosimioni** [reply]
Thank you @zmb3! I'll look into that.

**Reply by shikfu** [reply]
Is there a plan to support ldap?
